pkg/api/core/helper/common_test.go

```diff
@@ -10,8 +10,8 @@ import (
 	gomegatypes "github.com/onsi/gomega/types"
 	corev1 "k8s.io/api/core/v1"
 
+	. "github.com/gardener/gardener/pkg/api/core/helper"
```

Comment:
https://github.com/gardener/gardener/blob/7a923e1f5ee13335197a3e4cd7a0ee8543e0ef29/docs/development/changing-the-api.md#L30 has to be updated. You could also check for other references in the docs that need to be updated - I haven't check.